### PR TITLE
[Needs manual check] ppss.kr - Ad (with various types)

### PR DESCRIPTION
**Pull request by seia-soto**
This PR hides various types of ADs and thanks to ㅍㅍㅅㅅ site owner, we can safely mute and ignore AdShield on the site without getting the headache to remove it.

This filter won't remove adshield (as I am investigating their script) but would like to block the parent node of the ad and mute adshield not to tell `adblock detected`.

Only, I concern is concentrated to some mobile users as they don't have ability to inject scriptlets.

### CODE REVIEW

File: filters-AG/general_extension.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+ppss.kr#%#//scriptlet("set-constant", "Element.prototype.removeAttribute", "trueFunc")
```
Comment (seia-soto) [inline filters-AG/general_extension.txt:1]: This line prevents adshield to remove image tag properties such as `src`.

File: filters-AG/general_extension.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+ppss.kr#%#//scriptlet("set-constant", "alert", "trueFunc")
```
Comment (seia-soto) [inline filters-AG/general_extension.txt:2]: This line is used to mute adshield alerts.

File: filters-share/extended_css_ELEMHIDE.txt [hunk 3]
```diff
@@ -1,3 +1,9 @@
+ppss.kr###custom_html-3
+ppss.kr###genesis-content > div,span:not(:has(header)):not([itemprop])
```
Comment (seia-soto) [inline filters-share/extended_css_ELEMHIDE.txt]: This line blocks ad at top header.

File: filters-share/extended_css_ELEMHIDE.txt [hunk 4]
```diff
@@ -1,3 +1,9 @@
+ppss.kr###custom_html-3
```
Comment (seia-soto) [inline filters-share/extended_css_ELEMHIDE.txt]: Right side one with the `Recommended` title seems like only containing the ad.

File: filters-share/extended_css_ELEMHIDE.txt [hunk 5]
```diff
@@ -1,3 +1,9 @@
...
+ppss.kr###genesis-content > div,span:not(:has(header)):not([itemprop])
+ppss.kr###ppss_wing_banner_left
```
Comment (seia-soto) [inline filters-share/extended_css_ELEMHIDE.txt]: Left side one.

File: filters-share/extended_css_ELEMHIDE.txt [hunk 6]
```diff
@@ -1,3 +1,9 @@
...
+ppss.kr##.ad-video
```
Comment (seia-soto) [inline filters-share/extended_css_ELEMHIDE.txt]: Some YouTube embeds are popped-up later and embeds have parent node with `ad-video` class.

File: filters-share/extended_css_ELEMHIDE.txt [hunk 7]
```diff
@@ -1,3 +1,9 @@
...
+ppss.kr##.brand-margin
```
Comment (seia-soto) [inline filters-share/extended_css_ELEMHIDE.txt]: The class name doesn't look like ad but without this, we'll face huge external link.

File: filters-share/extended_css_ELEMHIDE.txt [hunk 8]
```diff
@@ -1,3 +1,9 @@
...
+ppss.kr##div[onclick*="clickSdrcCheck"]
```
Comment (seia-soto) [inline filters-share/extended_css_ELEMHIDE.txt]: This blocks all div tag with ad-related onclick event listeners.

File: filters-uBO/general_extension.txt [hunk 9]
```diff
@@ -1,3 +1,5 @@
+ppss.kr##+js(set, Element.prototype.removeAttribute, trueFunc)
+ppss.kr##+js(set, alert, trueFunc)
```
Comment (seia-soto) [inline filters-uBO/general_extension.txt:2]: uBO version.

**Comment by seia-soto** [comment]
Okay, some rules are blocking the site content on Chrome not like FF. Updated!

File: filters-share/specific_ELEMHIDE.txt [hunk 10]
```diff
@@ -1,3 +1,9 @@
+ppss.kr###custom_html-3
+ppss.kr###genesis-content > :not(.home-top, .post, .author-box, .pagination, .screen-reader-text)
```
Comment (seia-soto) [inline filters-share/specific_ELEMHIDE.txt:2]: New version of main content ad blocking rule.

**Review by piquark6046 — CHANGES_REQUESTED** [review]
It seems that your suggested rules break images of the website.

<details>
<summary>Screenshots</summary>

Screensho2
Screenshot1
Screenshot3


</details>

**Comment by seia-soto** [comment]
The original intent was blocking the ad nodes and muting the adshield but looks like this needs a lot of compatiblity checks. Skipping is better.

**Review by piquark6046 — APPROVED** [review]
Ad-Shield is blocked completely.
However, All images on the website are blocked incorrectly.

**Comment by seia-soto** [comment]
I think image is required to read the site content. Still Are images blocked?

**Comment by piquark6046** [comment]
https://github.com/List-KR/List-KR/pull/260#issuecomment-1077392292
Yeap.

**Comment by seia-soto** [comment]
The statement on my previous comment `Skipping is better` was it's better to skip merging. Please, revert as soon as possible. I am in investigation to reproduce your situation.

**Comment by piquark6046** [comment]
Okay.

**Pull request merged**